tests/filtered_relation/models.py

```diff
@@ -0,0 +1,109 @@
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+
+@python_2_unicode_compatible
+class Author(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+    favourite_books = models.ManyToManyField('Book',
```

Comment:
Hanging indent looks like this: 
```
favorite_books = models.ManyToManyField(
    'Book',
    related_name='preferred_by_authors',
    related_query_name='preferred_by_authors',
)
```
but probably it's better to try to reuse an existing test app/model rather than creating a new test app for this method.